app/views/show_url.erb

```diff
@@ -0,0 +1,75 @@
+<div id="sources">
+  <div class='col-md-12 navbar navbar-default navbar-static-top navbar-inverse'>
+    <a href='/' class='navbar-brand'>Rush Hour</a>
+    <ul class='nav navbar-nav navbar-right'>
+      <li><a href = "https://www.youtube.com/watch?v=ZFq_TFyhzmo">Full Movie</a></li>
+      <li><a href = "https://www.redbubble.com/shop/jackie+chan">Store</a></li>
+      <li class='dropwdown'>
+        <a href = '#' class='dropdown-toggle' data-toggle='dropdown'>Events
+        <span class="caret"></span></a>
+        <ul class='dropdown-menu'>
+          <% @events.each do |event| %>
+            <li><a href="/sources/<%= @client.identifier %>/events/<%= event %>"><%= event %></a></li>
+          <% end %>
+        </ul>
+      </li>
+    </ul>
+    <form class="navbar-form" role="search" action="/redirect" method="get">
+			<div class="input-group">
+				<input type="text" class="form-control" placeholder="Search" name="search-id">
+				<span class="input-group-btn">
+					<button type="submit" class="btn btn-default">
+						<span class="glyphicon glyphicon-search">
+							<span class="sr-only">Search</span>
+						</span>
+					</button>
+				</span>
+			</div>
+		</form>
+  </div>
+</div>
+
+<h1 class="title"> <%= @url.root_url + @url.path %> </h1>
```

Comment:
concatenating things is also too much logic in the view. we could have a method on our `Url` object that concatenates the `root_url` and the `path` to extract this out of the view